tools/functional-tester/etcd-tester/checks.go

```diff
@@ -80,27 +83,35 @@ func (hc *hashChecker) Check() error {
 type leaseChecker struct{ ls *leaseStresser }
 
 func (lc *leaseChecker) Check() error {
+	conn, err := grpc.Dial(lc.ls.endpoint, grpc.WithInsecure(), grpc.WithBackoffMaxDelay(1))
+	if err != nil {
+		return fmt.Errorf("%v (%s)", err, lc.ls.endpoint)
+	}
+	defer conn.Close()
+	kvc := pb.NewKVClient(conn)
+	leaseCli := pb.NewLeaseClient(conn)
```

Comment:
make these clients members of `leaseChecker` so they don't have to be passed around everywhere